challenge_serving_drinks/src/challenge_serving_drinks/serve_one_drink.py

```diff
@@ -0,0 +1,162 @@
+# ROS
+import smach
+
+# Robot smach states
+import robot_smach_states as states
+import robot_smach_states.util.designators as ds
+
+from robot_skills.util.entity import Entity
+from robocup_knowledge import knowledge_loader
+from robot_skills.robot import Robot
+
+# Serving drinks
+from .sd_states import DescriptionStrDesignator
+from .get_order import GetOrder
+
+# Knowledge
+CHALLENGE_KNOWLEDGE = knowledge_loader.load_knowledge("challenge_serving_drinks")
```

Comment:
Don't use capital letters, it is used for global constants.

Reply:
Will be reflected in the new PR (styling)